mzLib/mzLib.nuspec

```diff
@@ -23,6 +23,8 @@
         <dependency id="OpenMcdf" version="2.2.1.3" />
         <dependency id="OpenMcdf.Extensions" version="2.2.1.3" />
         <dependency id="SharpLearning.Optimization" version="0.28.0" />
+		<dependency id="CsvHelper" version="32.0.2" />
+		<dependency id="System.Data.SQLite" version="1.0.118" />
         <dependency id="Microsoft.Extensions.ObjectPool" version="9.0.0" />
```

Comment:
I trust you that we need to pack CsvHelper. I am curious as to why we need it now but did not previously